com.unity.render-pipelines.universal/Shaders/SpatialMappingOcclusion.shader

```diff
@@ -0,0 +1,78 @@
+Shader "Universal Render Pipeline/SpatialMappingOcclusion"
+{
+    Properties
+    {
+    }
+    SubShader
+    {
+        Tags {"RenderType" = "Opaque" "IgnoreProjector" = "False" "RenderPipeline" = "UniversalPipeline" "ShaderModel"="2.0"}
```

Comment:
Not sure about the implication of "ShaderModel" = "2.0" but that doesn't seem needed.